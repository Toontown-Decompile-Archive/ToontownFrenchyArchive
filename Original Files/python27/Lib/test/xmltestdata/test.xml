version https://git-lfs.github.com/spec/v1
oid sha256:c75afdd9fc3a44feb9900b8d6925d31f89b8830faac910be6658514a4c24d91b
size 1480
