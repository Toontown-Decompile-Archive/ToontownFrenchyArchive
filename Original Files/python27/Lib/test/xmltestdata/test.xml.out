version https://git-lfs.github.com/spec/v1
oid sha256:76799ffc2e90d73727ac88eba6317f27fee2b0b69a602fc1d9b3a8e162eb116a
size 1500
